controllers/openstack_ansibleee_controller.go

```diff
@@ -391,38 +424,103 @@ func addRoles(instance *redhatcomv1alpha1.OpenStackAnsibleEE, h *helper.Helper,
 	roleEnvVar.Value = "\n" + string(d) + "\n\n"
 	instance.Spec.Env = append(instance.Spec.Env, roleEnvVar)
 	job.Spec.Template.Spec.Containers[0].Env = instance.Spec.Env
+	hashes["roles"], err = calculateHash(string(d))
+	if err != nil {
+		fmt.Printf("Error calculating the hash!")
```

Comment:
here an below you should use the logger instead of printing to the standard out. 
```golang
h.GetLogger().Error(err, "Error calculating the hash")
```

Reply:
ack! I'll propose a follow up PR (I also need to fix the printing of a "label" that I used for debugging purposes)